db/re/item_db.conf

```
@@ -31808,10 +31808,16 @@ item_db: (
 	Name: "Rune Boots"
 	Type: "IT_ARMOR"
 	Buy: 20
-	Weight: 100
+	Weight: 10
```

```suggestion
	Weight: 100
```
I believe the weight is correct at `100` which is `10.0`. Your changes was wrong perhaps?

---

oh weight is `int`